styles/base.css

```diff
@@ -26,7 +26,7 @@ main {
 h1 {
   font-family: "Rakkas", cursive;
   font-size: 2em;
-  letter-spacing: 0.075em;
+  letter-spacing: 0.05em;
 }
 
 h2 {
```

Comment:
Ok, for whatever reason, I can't comment on the line I want to, but on line 67 where you set your min-width for the screen, There is a small amount of overlap and clipping that happens. Not a super big issue, but might be worth messing around with 👍